src/halo-hooks/useRewards.ts

```diff
@@ -307,3 +312,62 @@ export const useRewarderUSDPrice = (rewarderAddress: string | undefined) => {
 
   return rewarderTokenUsdPrice[rewarderTokenAddress[0]]
 }
+
+export const useGetBaseApr = (poolAddress: string, tokenPair: string): number => {
+  const { account, library, chainId } = useActiveWeb3React()
+  const [baseAPR, setBaseApr] = useState(0)
+  const poolsWithBaseAPR = Object.keys(DEPOSIT_TXHASH)
+
+  const fetchBaseUsdHlpPrice = useCallback(async () => {
```

Comment:
@geonellov should we move this calculation into its own calculator.ts function that we just import. this will make it easy for us to unit test different input/output

Reply:
@geonellov i tried calculating TCAD:USDC base apr manually and this is what i got

[First deposit tx](https://etherscan.io/tx/0x6038346288968a18c766987b8a7811a2df17a25316edeb23800bb2125ba5902f) -> got the given for `initial_USD_HLP_price` from here

Curve.totalSupply = 108713.205624799189781903
Curve.liquidity = 109066.176879507049362387

```
initial_USD_HLP_price = (95.25 + 94.93)/189.503999999999999999 = 1.003567207

current_USD_HLP_price = 109066.176879507049362387 / 108,713.205624799189781903 = 1.0032468112

Base APR = ((1.003567207 / 1.0032468112) - 1) * (365/208) * 100 ) = 0.05604134577 %
```